### PR TITLE
	new file:   css/bootstrap.css

### PR DESCRIPTION
**Pull request by simvyc**
new file:   css/style.css
	new file:   index.html
	new file:   js/bootstrapValidator.js
	new file:   js/js.js
	new file:   js/jumbotron.css
	new file:   pictures/background/background.jpg
	new file:   pictures/background/white_carbon.jpg
	new file:   pictures/countries/argentina.jpg
	new file:   pictures/countries/egypt.jpg
	new file:   pictures/countries/england.jpg
	new file:   pictures/countries/england1.jpg
	new file:   pictures/countries/estonia.jpg
	new file:   pictures/countries/france.jpg
	new file:   pictures/countries/france1.jpg
	new file:   pictures/countries/havaii.jpg
	new file:   pictures/countries/havaii1.jpg
	new file:   pictures/countries/latvia.jpg
	new file:   pictures/countries/lithuania.jpg
	new file:   pictures/countries/lithuania1.jpg
	new file:   pictures/countries/norway.jpg
	new file:   pictures/countries/peru.jpg
	new file:   pictures/countries/peru1.jpg
	new file:   pictures/countries/slovenia.jpg
	new file:   pictures/countries/switzerland.jpg
	new file:   pictures/countries/switzerland1.jpg
	new file:   pictures/countries/usa.jpg
	new file:   pictures/countries/usa1.jpg
	new file:   pictures/logo/logo1.png
	new file:   pictures/logo/logo3.png
	new file:   pictures/pexels_image_world.jpg
	new file:   templates/about.html
	new file:   templates/contact.html
	new file:   templates/lithuania.html
	new file:   templates/travels.html
	new file:   travel_web_page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel="icon" href="../CA_travel_web_page/pictures/logo/logo3.png">
+    <title>Simona's Travels</title>
+    <link href="css/bootstrap.css" rel="stylesheet">
+  </head>
+  <body background="../CA_travel_web_page/pictures/background/white_carbon.jpg">
```
Comment (MantsSk) [inline index.html]: Kodėl ne tiesiog href="pictures/background/white_carbon.jpg" ? 

index.html yra tom pačiam levelyje kaip ir pictures

File: index.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            <h2>Estonia</h2>
+            <img src="./pictures/countries/estonia.jpg" class="img-thumbnail">
+            <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
+            <p><a class="btn btn-info" href="../CA_travel_web_page/templates/contact.html" role="button">Contact us</a></p>
+          </div>
+          <div class="col-md-4">
+            <h2>Norway</h2>
+            <img src="./pictures/countries/norway.jpg" class="img-thumbnail">
+            <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
+            <p><a class="btn btn-info" href="../CA_travel_web_page/templates/contact.html" role="button">Contact us</a></p>
```
Comment (MantsSk) [inline index.html]: Kartojasi visur. Pakeičiant į "./templates/contact.html" veikia

**Review by MantsSk — CHANGES_REQUESTED** [review]
Matosi, kad daug darbo įdėta, tai šauniai tikrai :) Keleta pastabų dėl nuorodu turiu, keletas href'u neveikia

File: index.html (hunk 2 above)
Comment (simvyc) [reply index.html]: Ištaisiau

**Review by MantsSk — APPROVED** [review]
Good job 👍  Nepastebėjau kažkokių klaidų, gal kažkur nuorodų formatavimas galėtų vienos stilistikos laikytis, bet apart to viskas kaip ir good :)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+            <p><a class="btn btn-info" href="templates/contact.html" role="button">Contact us</a></p>
+          </div>
+          <div class="col-md-4">
+            <h2>Norway</h2>
+            <img src="pictures/countries/norway.jpg" class="img-thumbnail">
+            <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
+            <p><a class="btn btn-info" href="templates/contact.html" role="button">Contact us</a></p>
+        </div>
+        <hr>
+      </div>
```
Comment (VakarisZ) [inline index.html:115]: Truksta vieno `</div>`

File: templates/travels.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+              <th scope="row">5</th>
+              <td>Hawaii</td>
+              <td><img src="../pictures/countries/havaii1.jpg" class="img-fluid img-thumbnail" width="200" ></img></td>
+              <td>Summer, Autumn, Winter, Spring</td>
+              <td><a class="btn btn-info" href="../templates/contact.html" role="button">Go to Hawaii</a></td>
+            </tr>
+            <tr>
+              <th scope="row">6</th>
+              <td>France</td>
+              <td><img src="../pictures/countries/france1.jpg" class="img-fluid img-thumbnail" width="200" ></img></td>
```
Comment (VakarisZ) [inline templates/travels.html:92]: Kodel "width" neidejai i stiliu faila?

**Review by VakarisZ — COMMENTED** [review]
+ rep, kad panaudojai bootstrapa
- rep, kad mažokai CSS ir JS

File: templates/contact.html [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel="icon" href="../pictures/logo/logo3.png">
+    <title>Simona's Travels</title>
+    <link href="../css/bootstrap.css" rel="stylesheet">
+    <link href="../css/style.css" rel="stylesheet">
+  </head>
+  <body background="../pictures/background/background.jpg">
+    <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-info">
```
Comment (VakarisZ) [inline templates/contact.html:14]: Kodėl taip arti krašto? 
image

**Pull request merged**